amlip_cpp/test/manual/node/loop_computing.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright 2022 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file async_computing.cpp
+ *
+ */
+
+#include <algorithm>
+#include <thread>
+
+#include <cpp_utils/Log.hpp>
+
+#include <amlip_cpp/types/id/AmlipIdDataType.hpp>
+#include <amlip_cpp/node/ComputingNode.hpp>
+
+/*
+ * The job in this example will be a string serialized to bytes.
+ */
+int main(
+        int argc,
+        char** argv)
+{
+    // initialize the random number generator
+    srand (time(NULL));
+
+    // Activate log
+    eprosima::utils::Log::SetVerbosity(eprosima::utils::Log::Kind::Info);
+
+    logUser(AMLIPCPP_MANUAL_TEST, "Starting Manual Test Computing Node execution. Creating Node...");
+
+    while (1)
+    {
```

Comment:
This is indeed a cool opportunity to use the `SignalLogHandler` and make the process stop after a signal. The easiest way to do so is to cut this loop after a `process_job` has been called (we do not care if user waits for a bit) and a signal has been caught

Do the same in main